the-bureau/index.html

```diff
@@ -72,15 +86,17 @@ <h1 class="h3">Innovate</h1>
                     block__border-top
                     block__border-bottom">
         <h1 class="h2">Our core functions</h1>
-        <p>We work to give consumers the information they need to understand the terms of their agreements with financial companies. We are working to make regulations and guidance as clear and streamlined as possible so providers of consumer financial products and services can follow the rules on their own.</p>
-        
-        <div class="expandable 
-                    expandable__expanded 
-                    expandable__mobile-only 
-                    u-mb0 
+        <p>Congress created the CFPB to create a single point of accountability for enforcing
+        federal consumer financial laws and protecting consumers in the financial marketplace.
```

Comment:
Capitalize `federal`? The White House does it... http://www.whitehouse.gov/blog/2012/01/04/consumer-financial-protection-bureau-101-why-we-need-consumer-watchdog